### PR TITLE
fixed orca cli commands so that they won't fail playbook on empty res…

### PR DESCRIPTION
**Pull request by edenorca**
…ponse

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A simple fix that CLI commands will not fail playbooks on empty responses

## Screenshots
A simple fix that CLI commands will not fail playbooks on empty responses

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [X] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/Orca/Integrations/Orca/Orca.py [hunk 1]
```diff
@@ -219,20 +219,21 @@ def main() -> None:
...
-                                                      asset_unique_id=demisto_args.get('asset_unique_id'))
-            if isinstance(alerts, str):
-                return_error(alerts)
-            if not alerts:
-                return_error("Alerts not exists")
+            alert_type = demisto_args.get('alert_type')
+            asset_unique_id = demisto_args.get('asset_unique_id')
+            alerts = orca_client.get_alerts_by_filter(alert_type=alert_type, asset_unique_id=asset_unique_id)
+            if not alerts or isinstance(alerts, str):
+                error_message = f"Could not find alerts from {alert_type=}" if alert_type else f"Could not find alerts from {asset_unique_id=}"  # noqa: E501
```
Comment (barchen1) [inline Packs/Orca/Integrations/Orca/Orca.py:226]: It is better to return this error in the human-readable instead of showing this error in the context.
also, please remove `# noqa: E501` and fix the error instead of ignoring it.

File: Packs/Orca/Integrations/Orca/Orca.py [hunk 2]
```diff
@@ -219,20 +219,21 @@ def main() -> None:
...
+                alerts = {"alerts": error_message}  # type: ignore
```
Comment (barchen1) [inline Packs/Orca/Integrations/Orca/Orca.py:227]: why did you add `type: ignore`? 
I think we should return an empty list instead of the error message

File: Packs/Orca/Integrations/Orca/Orca.py [hunk 3]
```diff
@@ -219,20 +219,21 @@ def main() -> None:
...
+
             command_result = CommandResults(outputs_prefix="Orca.Manager.Alerts", outputs=alerts, raw_response=alerts)
             return_results(command_result)
 
         elif command == "orca-get-asset":
             asset = orca_client.get_asset(asset_unique_id=demisto.args()['asset_unique_id'])
             if not isinstance(asset, Dict):
                 # this means asset not found
-                return_error(asset)
+                asset = {"asset": asset}  # type: ignore
```
Comment (barchen1) [inline Packs/Orca/Integrations/Orca/Orca.py:236]: same as above

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution!
Please fix the comments I gave you

Let me know if you have any questions

File: Packs/Orca/ReleaseNotes/1_0_2.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Orca
+- Changed Orca CLI commands so that they will not fail playbooks on empty result
```
[the remark this answers is not in the document]
Comment (edenorca) [reply Packs/Orca/ReleaseNotes/1_0_2.md:4]: done

File: Packs/Orca/Integrations/Orca/Orca.py (hunk 3 above)
Comment (edenorca) [reply Packs/Orca/Integrations/Orca/Orca.py:236]: done

File: Packs/Orca/Integrations/Orca/Orca.py (hunk 2 above)
Comment (edenorca) [reply Packs/Orca/Integrations/Orca/Orca.py:227]: done

**Review by guykeller — APPROVED** [review]

**Review by barchen1 — APPROVED** [review]
